client/src/components/SubstraitGraph.vue

```diff
@@ -85,6 +99,16 @@ export default {
       };
       image.src = svgUrl;
     },
+    generateJSON() {
+      const jsonObject = JSON.parse(store.plan);
+      const jsonString = JSON.stringify(jsonObject, null, 4);
```

Comment:
should we define `4` with a const variable and a proper definition for it (a comment would be nice)

Reply:
As discussed in PR #27, using `const` here will not be of much help. 
Commented on the usage of the `JSON.stringify()` parameters.